package/skelcd-control-suse-manager-server.changes

```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Mon Nov  7 12:32:34 UTC 2022 - Julio González Gil <jgonzalez@suse.com>
+
+- SUSE Manager version 4.4.0
```

Comment:
Please add a Bugzilla (`bsc#`) or Jira (`jsc#`) reference. That's required by the autosubmission check in Jenkins.